src/main/java/com/myhome/services/springdatajpa/UserSDJpaService.java

```diff
@@ -69,19 +69,19 @@
   @Override
   public Optional<UserDto> getUserDetails(String userId) {
     User user = userRepository.findByUserId(userId);
+    if(user != null) {
+      Set<String> communityIds = communityService.listAll().stream().filter(c -> c.getAdmins()
```

Comment:
If you are in this class it would be great if you can change some formatting and naming as suggested below:
```
    Set<String> communityIds = communityService.listAll().stream()
        .filter(community -> community.getAdmins()
            .stream()
            .map(CommunityAdmin::getAdminId)
            .collect(Collectors.toSet())
            .contains(userId)
        )
        .map(Community::getCommunityId)
        .collect(Collectors.toSet());
```